xbmc/music/MusicDatabase.cpp

```diff
@@ -9288,6 +9415,9 @@ void CMusicDatabase::UpdateTables(int version)
     m_pDS->exec("DROP TABLE artist");
     m_pDS->exec("ALTER TABLE artist_new RENAME TO artist");
   }
+
+  if (version < 83)
+    m_pDS->exec("ALTER TABLE song ADD strVideoURL TEXT");
```

Comment:
Maybe you want to add a blank line after this line to increase readability of the code.

Reply:
Done.